File: Java/DPR.java
public class DPR {

    private String name;
    private String partai;
    private String jabatan;
    private String komisi;

    public DPR() {
        this.name = "";
        this.partai = "";
        this.jabatan = "";
        this.komisi = "";
    }

    public DPR(String name, String partai, String jabatan, String komisi) {
        this.name = name;
        this.partai = partai;
        this.jabatan = jabatan;
        this.komisi = komisi;
    }

    // mengambil dan mengatur untuk name
    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    // mengambil dan mengatur untuk partai
    public String getPartai() {
        return this.partai;
    }

    public void setPartai(String partai) {
        this.partai = partai;
    }

    // mengambil dan mengatur untuk jabatan
    public String getJabatan() {
        return this.jabatan;
    }

    public void setJabatan(String jabatan) {
        this.jabatan = jabatan;
    }

    // mengambil dan mengatur untuk komisi
    public String getKomisi() {
        return this.komisi;
    }

    public void setKomisi(String komisi) {
        this.komisi = komisi;
    }
}

File: Python/DPR.py
class DPR:
    __name = ""
    __partai = ""
    __jabatan = ""
    __komisi = ""
    

    def __init__(self, name = "", partai = "", jabatan = "", komisi = ""):
        self.__name = name
        self.__partai = partai
        self.__jabatan = jabatan
        self.__komisi = komisi

    # Untuk Get dan Set pada nama
    def get_name(self):
        return self.__name
    
    def set_name(self, name):
        self.__name = name

    # Untuk Get dan Set Pada partai
    def get_partai(self):
        return self.__partai

    def set_partai(self, partai):
        self.__partai = partai

    # Untuk Get dan Set Pada jabatan
    def get_jabatan(self):
        return self.__jabatan

    def set_jabatan(self, jabatan):
        self.__jabatan = jabatan

    # Untuk Get dan Set Pada komisi
    def get_komisi(self):
        return self.__komisi

    def set_komisi(self, komisi):
        self.__komisi = komisi


